src/OrleansGCPUtils/OrleansGCPUtils.csproj

```diff
@@ -0,0 +1,69 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{6E5860C5-44E7-415C-80D6-3ECF15A80796}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>OrleansGCPUtils</RootNamespace>
+    <AssemblyName>OrleansGCPUtils</AssemblyName>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <TargetFrameworkProfile />
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="GoogleErrorCodes.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubAdapter.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubAdapterConstants.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubAdapterFactory.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubAdapterReceiver.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubBatchContainer.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubDataManager.cs" />
+    <Compile Include="Providers\Streams\PubSub\PubSubStreamProvider.cs" />
+    <Compile Include="Providers\Streams\PubSub\IPubSubDataAdapter.cs" />
```

Comment:
The reason it was put this way, is to follow the most complete stream implementation which is EventHubs and also follow @gabikliot advices regarding use the proper naming of the product. Also, keep in mind that it is one (the most basic) Stream provider for Google GCP. I plan to have later a more feature full one that uses Google PubSubIO, the same way we did with Azure Queue -> Event Hubs and that new provider would be under `Providers\Streams\PubSubIO`.